src/scanner-cli.ts

```diff
@@ -46,6 +47,10 @@ export function normalizePlatformName(): 'windows' | 'linux' | 'macosx' {
 export async function tryLocalSonarScannerExecutable(command: string): Promise<boolean> {
   return new Promise<boolean>(resolve => {
     log(LogLevel.INFO, `Trying to find a local install of the SonarScanner: ${command}`);
+    if (!fsExtra.existsSync(command)) {
```

Comment:
this will not allow for PATH resolution, right?

Reply:
related issue
https://cirrus-ci.com/task/4586583599349760?logs=sonarcloud_analysis#L37